client/screens/HomeScreen.js

```diff
@@ -65,28 +66,21 @@ const HomeScreen = props => {
   const [password, setPassword] = useState('')
 
   const loginUser = async () => {
-    const userData = {email: 'cc', leaves: 0, name: 'cc'}
-    await AsyncStorage.setItem('LOGGED_IN_USER', JSON.stringify(userData))
-    navigate('Snap', userData)
     const {client, navigation} = props
     if ([email, password].every(i => i && i.trim())) {
       try {
-        const userData = {email: 'cc', leaves: 0, name: 'cc'}
-        await AsyncStorage.setItem('LOGGED_IN_USER', JSON.stringify(userData))
-        navigation.navigate('Snap', userData)
         const result = await client.query({
           query: CHECK_USER_EXISTS,
           variables: {
             email,
             password
           }
         })
-
-        // const userData = result.data.user
-        // await AsyncStorage.setItem('LOGGED_IN_USER', JSON.stringify(userData))
-        // navigation.navigate('Snap', userData)
+        const userData = result.data.user
+        await AsyncStorage.setItem('LOGGED_IN_USER', JSON.stringify(userData))
+        navigation.navigate('Snap', userData)
       } catch (error) {
-        console.log('errrrr', error)
+        console.log('errrrr', JSON.stringify(error))
```

Comment:
I think the best way to console log error is to use built in method: 
`console.error(error)`
In that manner you got in console:
- error message
- stacktrace